.github/workflows/ci.yaml

```diff
@@ -33,9 +33,9 @@ jobs:
       fail-fast: false
       matrix:
         juju:
-          - agent: 2.9.46
-            libjuju: ^2
-          - agent: 3.1.7
+          - agent: 2.9.46  # renovate: latest juju 2
+            libjuju: ==2.9.46.1  # renovate: latest libjuju 2
```

Comment:
Currently we install the latest libjuju 2. Do we want to pin?

Reply:
If we pin it, does Renovate update it in its PRs?

Reply:
Yes, Renovate will update the Juju 2 agent and libjuju 2 in the same PR.